docs/workforce-management-framework/topics/con-workflow-step.adoc

```diff
@@ -0,0 +1,60 @@
+= Workflow Step
+
+== Step Structure
+Steps are divided into to linked catergories the `module` and the `definition`.
+
+=== Module
+Modules can contain one or more angular directives for rendering steps in the portal and mobile applications. 
```

Comment:
Change "in the portal and mobile applications. " to "in the Client App (mobile) and the Cloud App."

Reply:
@jstaffor why we changing that?

Reply:
@wtrocki - this is more inline with other documentation. (If it's wrong to say this, no probs leaving it as is).

"The {WFM-RC-NameShort} Datasync Module is an implementation of the {WFM-RC-NameShort} Sync Framework. Datasync is made up of two applications, the Cloud App and the Client App."[1]


[1] https://github.com/feedhenry-raincatcher/raincatcher-docs/blob/master/docs/workforce-management-framework/topics/con-datasync-module.adoc

Reply:
That makes much more sense now @wtrocki :+1: 

Given the original:

"Modules can contain one or more angular directives for rendering steps in the portal and mobile applications."

Which of the following is most applicable:

1) "Modules can contain one or more angular directives for rendering steps in the Portal and the Client App (mobile)."
2) "Modules can contain one or more angular directives for rendering steps in the Portal, the Client App (mobile) and the Cloud App." (think local development where I have both running locally (I could be over complicating this - call it out if so))

Reply:
I think the confusion here is that the feedhenry sync documentation refers to client and cloud.  Portal , mobile are what they are called in the platform , I think we should match the platforms naming conventions. So leaving as is.

Reply:
@jstaffor - I think @austincunningham added some foto to your comment. 
Our names are now as above. That's are short names. 
Long names (suggestion) are posted on our main website.